product_unique_serial/wizard/stock_serial.py

```diff
@@ -156,47 +174,18 @@ class StockSerialLine(models.TransientModel):
 
     _name = 'stock.serial.line'
 
+    @api.multi
+    @api.depends('serial')
+    def _compute_lot_id(self):
```

Comment:
you right! about give a warning to user using onchange, i will bring it back this functionality... but what you think about computed field? left only the onchange function?